src/encoded/types/base.py

```diff
@@ -66,18 +67,20 @@ def __init__(self, *args, **kw):
         super(AbstractCollection, self).__init__(*args, **kw)
 
     def get(self, name, default=None):
-        """
-        heres' and example of why this is the way it is:
-        ontology terms have uuid or term_id as unique ID keys
-        and if neither of those are included in post, try to
-        use term_name such that:
-        No - fail load with non-existing term message
-        Multiple - fail load with ‘ambiguous name - more than 1 term with that name exist use ID’
-        Single result - get uuid and use that for post/patch
+        """Override method to allow more lookup keys for items.
+
+        Base method allows lookup by uuid and unique key, but other
+        unique keys can also be used, such as accession or aliases.
         """
         resource = super(AbstractCollection, self).get(name, None)
         if resource is not None:
             return resource
+        if is_accession(name):
+            resource = self.connection.get_by_unique_key('accession', name)
+            if resource is not None:
+                if not self._allow_contained(resource):
+                    return default
+                return resource
```

Comment:
Did you find this to be necessary? I think L75 should handle this as it calls out to https://github.com/4dn-dcic/snovault/blob/master/snovault/resources.py#L309-L317 no? If this doesn't work, I think it means the unique key is not specified correctly?

Reply:
This is helpful to get over the limitation of the designation of only one unique key, limiting lookups with the collection (e.g. `/file/.../`) to either the UUID or the unique key. For example, for all submitted files, the unique key is the `submitted_id`, and that means accessions currently do not work when provided with the collection. The above fixes this to allow accessions, which should always work given the accession alone is able to redirect to the item.

More importantly, during submission with submitr links are formatted into the collection-style lookup (`/file/.../`), so accessions are not able to be used currently for submitted items and other item types that have accessions but non-accession unique keys. With the change above, accessions are usable, as well as UUIDs and designated unique keys.